### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by slowpokex**

### CODE REVIEW

**Comment by NikitaSedyx** [comment]
what is conflict? it's your the first PR, please do not merge anything directly to master

**Comment by NikitaSedyx** [comment]
is task completed? I don't see changes related to it

**Comment by slowpokex** [comment]
I hope that this is what you need! Because I at first did not quite understand the logic of pull request

**Comment by slowpokex** [comment]
Can this option suit?

File: css/main.css [hunk 1]
```diff
@@ -4,59 +4,77 @@ body {
...
-.header h2 {
-  font-family: sans-serif;
-  font-style: italic;
+  display: inline-block;
 }
 
 .content {
-    position: inherit;
-    height: 500px;
+    padding: 5% 0 5% 0;
```
Comment (NikitaSedyx) [inline css/main.css]: !

File: css/main.css [hunk 2]
```diff
@@ -4,59 +4,77 @@ body {
...
+    overflow: hidden;
 }
 
-.blocks.block {
-
+.block {
+    position: inherit;
+    width: 150px;
+    height: 150px;
+    margin: 30px 30px 30px 30px;
```
Comment (NikitaSedyx) [inline css/main.css]: !

File: css/main.css [hunk 3]
```diff
@@ -4,59 +4,77 @@ body {
...
-  flex-direction: row;
-  justify-content: space-around;
 }
 
 .head_element {
-  position: relative;
-  border: 1px solid;
+  position: inherit;
   margin: 5px 5px 5px 5px;
+  padding: 5px 5px 5px 5px;
```
Comment (NikitaSedyx) [inline css/main.css]: !

File: css/main.css [hunk 4]
```diff
@@ -4,59 +4,77 @@ body {
 }
 
 .header {
+  position: relative;
```
Comment (NikitaSedyx) [inline css/main.css]: do you really need this position here?

File: css/main.css [hunk 5]
```diff
@@ -4,59 +4,77 @@ body {
...
     border-top: 1px solid;
     border-bottom: 2px solid;
     width: 100%;
     height: 40px;
+    text-align: center;
+    background-color: #9bff8a;
+}
+
+.footer_element {
+    margin: 5px 5px 5px 5px;
```
Comment (NikitaSedyx) [inline css/main.css]: !

**Pull request closed without merging**